quartz/components/ContentMeta.tsx

```
@@ -1,18 +1,22 @@
-import { formatDate, getDate } from "./Date"
+import { Date, getDate } from "./Date"
```

Biome (2.1.2)

[error] Do not shadow the global "Date" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)